backend/scripts/load_gerrydb_view.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I am very stupid, where is this invoked?

Reply:
Just locally for now. We should ask if there's a need for batch imports like I'm doing here on prod. For the time being I rather not add a command to do this, requiring admins to load layers 1 by 1.